src/System.Net.Security/src/System/Net/CertificateValidationPal.Unix.cs

```diff
@@ -246,7 +246,8 @@ private static int QueryContextRemoteCertificate(SafeDeleteContext securityConte
 
         private static int QueryContextIssuerList(SafeDeleteContext securityContext, out Object issuerList)
         {
-            // TODO (Issue #3362) To be implemented
+            // Code to get the Issuer List is in GetRequestCertificateAuthorities()
+            // Being a PAL function , this needs to be present but not used anywhere.
```

Comment:
This is the only "QueryContextIssuerList" I see being mentioned anywhere in all of corefx.  Can this function just be deleted entirely?